<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Rin Icon.png</normaloff>Rin Icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btn_Check_Feed">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Checks the subscription_manager.xml file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Check Channel Feed [Ctril + 1]</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+1</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Update_Date">
           <property name="toolTip">
            <string>Updates the Last Checked Date to today</string>
           </property>
           <property name="text">
            <string>Update Date [Ctrl + 2]</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Watch_Later">
           <property name="toolTip">
            <string>Adds the videos below to your watch later list</string>
           </property>
           <property name="text">
            <string>Add to Watch Later [Ctrl + 3]</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="pb_adding_videos">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Progress for Adding Videos to Watch Later Playlist</string>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Last Checked Date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="last_checked_date">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="displayFormat">
            <string>MM/dd/yyyy</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTreeView" name="trv_Video_Feed_View">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="headerHidden">
         <bool>false</bool>
        </property>
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>500</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="act_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit  [Ctrl+Shift+Q]</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="act_Update_Date">
   <property name="text">
    <string>Update Date</string>
   </property>
   <property name="toolTip">
    <string>Update Date  [Ctrl+2]</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="act_Check_Feed">
   <property name="text">
    <string>Check Feed</string>
   </property>
   <property name="toolTip">
    <string>Check Feed [Ctrl+1]</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="act_Watch_Later">
   <property name="text">
    <string>Add to Watch Later</string>
   </property>
   <property name="toolTip">
    <string>Add to Watch Later  [Ctrl+3]</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="act_Add_to_Playlist">
   <property name="text">
    <string>Add to Other Playlist</string>
   </property>
   <property name="toolTip">
    <string>Add to Other Playlist  [Ctrl+4]</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>last_checked_date</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
